install/chrome_extensions/install-ajktown-recommmended-chrome-extensions.md

```diff
@@ -0,0 +1,18 @@
+# Install AJK Town Recommended Chrome Extensions
+
+<!-- TOC -->
+
+- [Install AJK Town Recommended Chrome Extensions](#install-ajk-town-recommended-chrome-extensions)
+  - [Overview](#overview)
+    - [JSON formatter](#json-formatter)
+
+<!-- /TOC -->
+
+## Overview
+
+This doc is to install the recommended extensions for their chrome to see json data from API server
```

Comment:
This entire doc is not only for the json (Although we have only json formatter at this point)

```suggestion
This doc is to install the recommended extensions for AJK Town for your Chrome browser
```